pkgs/tools/security/nitrokey-app2/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib
+, fetchFromGitHub
+, fetchpatch
+, wrapQtAppsHook
+, python3
+, pynitrokey
+}:
+
+let
+  python' = python3.override {
+    self = python';
+    packageOverrides = final: prev: {
+      pyqt5 = prev.pyqt5.override {
+        withConnectivity = true;
+        withMultimedia = true;
+        withLocation = true;
+        withSerialPort = true;
+        withTools = true;
```

Comment:
We should move a full pyqt5 into python-packages otherwise we need to compile two just because another package is slightly different which is rather wasteful https://github.com/NixOS/nixpkgs/blob/5bb032cb23965aa6bb05d2e8327a71eb7fca0251/pkgs/development/python-modules/qtpy/default.nix#L35